examples/src/custom_mpl_cmm_example.py

```diff
@@ -0,0 +1,117 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Example to create a custom implementation of the ESDK-MPL ICryptographicMaterialsManager class.
```

Comment:
```suggestion
Example to create a custom implementation of the MPL's ICryptographicMaterialsManager class
and use it with the ESDK.
```

I like where your head was at here, but it really is just the MPL ICMM.
I think this better communicates that the interface is for the MPL (implicitly, can be used in DBESDK + S3EC once those are out)
and says that this is just using it with ESDK

Reply:
That makes sense. Thanks.